src/coprocessor/tracker.rs

```diff
@@ -203,6 +210,8 @@ impl Tracker {
         let mut detail_v2 = ScanDetailV2::default();
         detail_v2.set_processed_versions(self.total_storage_stats.write.processed_keys as u64);
         detail_v2.set_total_versions(self.total_storage_stats.write.total_op_count() as u64);
+        let read_bytes = self.total_storage_stats.write.flow_stats.read_bytes+self.total_storage_stats.data.flow_stats.read_bytes;
```

Comment:
Should also add `self.total_storage_stats.lock.flow_stats.read_bytes` ?

Reply:
Agree. 

Furthermore, you can introduce a `sum`/`total` function for `Statistics`; maybe like `pub fn sum(&self) -> CfStatistics`.

Then we get:
```rust
let read_bytes = self.total_storage_stats.sum().flow_stats.read_bytes;
```